internal/utils/min_max_avx2_amd64.s

```diff
@@ -488,7 +488,7 @@ LBB3_5:
 DATA LCDATA3<>+0x000(SB)/8, $0x7fffffff80000000
 GLOBL LCDATA3<>(SB), 8, $8
 
-TEXT ·_int32_max_min_avx2(SB), $0-32
+TEXT ·_int32_max_min_avx2(SB), NOSPLIT, $8-32
```

Comment:
Is the NOSPLIT only needed here and not in the other locations?

Reply:
ah, I can probably add it to the other locations for the benefits